apps/directory/src/stores/filtersStore.ts

```diff
@@ -27,11 +29,11 @@ export const useFiltersStore = defineStore("filtersStore", () => {
   /** check for filter manipulations */
   let filterTriggeredBookmark = ref(false);
 
-  let filters = ref({});
-  let filterType = ref({});
-  let filterOptionsCache = ref({});
-  let filterFacets = ref([]);
-  const facetDetailsDictionary = ref({});
+  let filters = ref<Record<string, any>>({});
+  let filterType = ref<Record<string, any>>({});
+  let filterOptionsCache = ref<Record<string, any>>({});
+  let filterFacets = ref<any[]>([]);
+  const facetDetailsDictionary = ref<Record<string, any>>({});
```

Comment:
most of the time refs can be const as usually the value is changhed and not the object

Reply:
I know, but that's not for this pr. I personally prefer let anyway to indicating it's something that changes internally.